src/components/Tables/ResourceAvailabilityTable.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div>
+    <div v-if="resources.length === 0">
+      <p class="no-resources">No resources available.</p>
+    </div>
+    <div v-else>
+      <div class="table">
+        <table class="table-table">
+          <thead class="table-component">
+            <tr>
+              <th>Owner Name</th>
```

Comment:
The owner name would be the same for every resource in this table because it's showing us the resources a certain user possess. That means we don't need to have it on the table rows. It might be a good idea to include it in the table header though.

Reply:
Done